test/skimNtuple2016_HHbtag.cpp

```diff
@@ -0,0 +1,6078 @@
+
+        // Get HHbtag scores in a map <jet_idx, HHbtag_score>
+        std::map<int,float> jets_and_HHbtag = HHbtagTagger.GetScore();
+
+        // Use a duplicated map to get the two highest values
+        // through get_max_map() from skimUtils.h
+        std::map<int,float> HHbtag_idx_map = jets_and_HHbtag;
+        auto HHbtagmax  = get_max_map(HHbtag_idx_map);
+        HHbtag_idx_map.erase(HHbtagmax.first);
+        auto HHbtagmax2 = get_max_map(HHbtag_idx_map);
+
+        if (DEBUG)
+        {
+          std::cout << "-------------- jets_and_HHbtag --------------" << std::endl;
+          for (auto elem : jets_and_HHbtag)
+          {
+            std::cout << "  - idx: " << elem.first << " / value: " << elem.second << std::endl;
+          }
+          std::cout << "  B1 idx: " << HHbtagmax.first  << " value: " << HHbtagmax.second  << std::endl;
+          std::cout << "  B2 idx: " << HHbtagmax2.first << " value: " << HHbtagmax2.second << std::endl;
+          std::cout << "---------------------------------------------" << std::endl;
+        }
+
+        // bjet1 and bjet2 indexes
+        const int bjet1idx = HHbtagmax .first ;
+        const int bjet2idx = HHbtagmax2.first ;
+
+        // Check that both bjets are assigned
+        if (bjet1idx < 0)
+        {
+          std::cout << "**ERROR: HHbtag score not found for bjet1 --> skipping event !!" << std::endl;
+          continue;
+        }
+        if (bjet2idx < 0)
+        {
+          std::cout << "**ERROR: HHbtag score not found for bjet2 --> skipping event !!" << std::endl;
+          continue;
+        }
+
+        //bool bPairFound = false;
+        //int njets = jets_and_sortPar.size();
+
+        // medium WP for 2016 is: 0.6321 for DeepCSV, 0.3093 for DeepFlavor
+        //if(useDeepFlavor)
+        //{
+        //    if (jets_and_sortPar.at(njets-2).first>0.3093) bPairFound = true;
+        //}
+        //else
+        //{
+        //    if (jets_and_sortPar.at(njets-2).first>0.6321) bPairFound = true;
+        //}
+
+        //const int bjet1idx = jets_and_sortPar.at(njets-1).second ;
+        //int bjet2idx_temp  = jets_and_sortPar.at(njets-2).second ;
+
+        //VBF tag
+        int VBFidx1 = -1;
+        int VBFidx2 = -1;
+        theSmallTree.m_isVBF = 0;
+
+        // build all the possible VBF-jet pairs
+        std::vector< tuple<float, int, int> > VBFcand_Mjj;
+        if (theBigTree.jets_px->size ()>1)
+        {
+          for (unsigned int iJet = 0; (iJet < theBigTree.jets_px->size ()) && (theSmallTree.m_njets < maxNjetsSaved); ++iJet)
+          {
+            // JET PF ID cut
+            if (theBigTree.PFjetID->at (iJet) < PFjetID_WP) continue; // 0 ; don't pass PF Jet ID; 1: tight, 2: tightLepVeto
+
+            // Skip the already selected b-jets
+            if (int (iJet) == bjet1idx) continue;
+            if (int (iJet) == bjet2idx) continue;
+            //if(bPairFound && int (iJet) == bjet2idx_temp) continue;
```

Comment:
this commented line is probably just useless

Reply:
ok! I will remove the useless commented lines.